dart/content.md

```diff
@@ -0,0 +1,60 @@
+# What is Dart?
+
+Dart is a client-optimized language for developing fast apps on any platform. Its goal is to offer the most productive programming language for multi-platform development, paired with a flexible execution runtime platform for app frameworks. For more details, see https://dart.dev.
+
+By utilizing Dart's support for ahead-of-time (AOT) [compilation to executables](https://dart.dev/tools/dart-compile#exe), you can create very small runtime images (~10 MB).
+
+%%LOGO%%
+
+# Quick reference
+
+-	**Where to get help**: [the Docker Community Forums](https://forums.docker.com/), [the Docker Community Slack](https://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker)
```

Comment:
The templates will generate a "Quick Reference" section at the top of the `README.md` and will have this same bullet point (with the same content).

Reply:
Thanks, removed.